src/BaselinePerIdentifierFormatter.php

```diff
@@ -29,8 +29,12 @@ class BaselinePerIdentifierFormatter implements ErrorFormatter
 
     private string $indent;
 
-    public function __construct(string $baselinesDir, string $indent)
+    public function __construct(?string $baselinesDir, string $indent)
     {
+        if ($baselinesDir === null) {
+            throw new LogicException('Baselines directory must be set, please set up \'parameters.shipmonkBaselinePerIdentifier.directory\' in your phpstan.neon.dist');
```

Comment:
```suggestion
            throw new LogicException('Baselines directory must be set, please set up \'parameters.shipmonkBaselinePerIdentifier.directory\' in your phpstan configuration file');
```

Not everyone uses phpstan.neon.dist, it could also be phpstan.neon or phpstan.php.

Reply:
Adjusted